logging/bin/deploy_eventrouter.sh

```diff
@@ -10,6 +10,21 @@ this_script=`basename "$0"`
 
 log_debug "Script [$this_script] has started [$(date)]"
 
+eventrouterRegistry="gcr.io"
+eventrouterPullSecret="null"
+
+## Check for air gap deployment
+if [ "$AIRGAP_DEPLOYMENT" == "true" ]; then
+  
+  # Check for the image pull secret for the air gap environment
+  checkForAirgapSecretToNamespace "$AIRGAP_IMAGE_PULL_SECRET_NAME" "$LOG_NS"
+
+  eventrouterRegistry=$AIRGAP_REGISTRY
+  eventrouterPullSecret=$AIRGAP_IMAGE_PULL_SECRET_NAME
```

Comment:
Because the Event Router file is particular and doesn't have an additional airgap YAML file that we're adding onto it, it was the only way that I could think of to make sure that the registry and the pull secret were set to "gcr.io" and "null," respectively in the case where we were not doing an air gap deployment.

Reply:
ok